src/components/tables/Table.tsx

```diff
@@ -210,5 +210,9 @@ export class Table extends React.Component<ITableProps, {}> {
                 />
             );
         });
+
+        return this.props.collapsibleFormatter
+            ? tableBodyNode
+            : <tbody>{tableBodyNode}</tbody>;

```

Comment:
I would suggest adding a comment here to explain the situation since it is a peculiar case